ci/aws/test-sr-iov-secondary-network-aws.sh

```diff
@@ -0,0 +1,354 @@
+        kubectl get nodes
+EOF
+}
+
+function setup_cluster {
+    # Launch Master and Worker EC2 instances
+    echo "Launching EC2 instance ..."
+    MASTER_INSTANCE_ID=$(launch_ec2_instance "ControlPlane")
+    echo "ControlPlane EC2 instance launched with Instance ID: $MASTER_INSTANCE_ID"
+    WORKER_INSTANCE_ID=$(launch_ec2_instance "Worker")
+    echo "Worker EC2 instance launched with Instance ID: $WORKER_INSTANCE_ID"
+
+    attach_network_interface "$MASTER_INSTANCE_ID"
+    attach_network_interface "$WORKER_INSTANCE_ID"
+
+    echo "========MASTER_INSTANCE_ID: $MASTER_INSTANCE_ID, MASTER_INSTANCE_ID: $WORKER_INSTANCE_ID========="
+
+    # Get the public IP addresses of the instances
+    MASTER_IP=$(get_instance_ip "$MASTER_INSTANCE_ID")
+    echo "Get PublicIpAddress of EC2 Instance $MASTER_INSTANCE_ID: $MASTER_IP"
+    WORKER_IP=$(get_instance_ip "$WORKER_INSTANCE_ID")
+    echo "Get PublicIpAddress of EC2 Instance $WORKER_INSTANCE_ID: $WORKER_IP"
+
+    echo "======MASTER_IP: $MASTER_IP, WORKER_IP: $WORKER_IP==========="
+
+    # Wait for EC2 instances to be fully ready
+    echo "Waiting for EC2 instances to be ready..."
+    sleep 120  # You may adjust the sleep time based on your environment
+
+    # Install Kubernetes on both Master and Worker nodes
+    install_kubernetes "$MASTER_IP"
+    install_kubernetes "$WORKER_IP"
+
+    # Initialize Kubernetes master node, if it fails, it will not exit
+    initialize_master "$MASTER_IP" || {
+        echo "Failed to initialize master node. Re-running installation for both nodes."
+        # Re-run the install commands for both master and worker nodes
+        install_kubernetes "$MASTER_IP"
+        install_kubernetes "$WORKER_IP"
+        # After re-installing, attempt the initialization again
+        initialize_master "$MASTER_IP" || {
+            echo "Initialization ControlPlane failed again, please check logs."
+            exit 1
+        }
+    }
+
+    # Get the join command and join the worker node to the cluster
+    JOIN_COMMAND=$(get_join_command "$MASTER_IP")
+    join_worker "$WORKER_IP" "$JOIN_COMMAND"
+
+    # Verify the Kubernetes cluster status
+    verify_cluster "$MASTER_IP"
+    
+    echo "Kubernetes cluster setup completed!"
+}
+
+build_image() {
+    # Ensure that files in the Docker context have the correct permissions, or Docker caching cannot
+    # be leveraged successfully
+    chmod -R g-w build/images/ovs
+    chmod -R g-w build/images/base
+    # Pull images from Dockerhub first then try Harbor.
+    for i in `seq 3`; do
+        ./hack/build-antrea-linux-all.sh --pull && break
+    done
+    if [ $? -ne 0 ]; then
+        echoerr "Failed to build antrea images with Dockerhub"
+        for i in `seq 3`; do
+            DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull && break
+        done
+        if [ $? -ne 0 ]; then
+            echoerr "Failed to build antrea images with Harbor"
+            exit 1
+        fi
+    fi
```

Comment:
These are unnecessary, the DOCKER_REGISTRY in the ci/docker-registry is always docker.io now, we don't maintain the Harbor images anymore.

Reply:
updated.